tekton/utils_test.go

```diff
@@ -0,0 +1,100 @@
+/*
+Copyright 2022 Red Hat Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package tekton
+
+import (
+	"context"
+	"reflect"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+
+	"github.com/redhat-appstudio/release-service/api/v1alpha1"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Utils", func() {
+
+	const (
+		pipelineRunPrefixName = "test-pipeline"
+		apiVersion            = "appstudio.redhat.com/v1alpha1"
+		namespace             = "default"
+	)
+
+	var release *v1alpha1.Release
+	var releasePipelineRun *ReleasePipelineRun
+
+	BeforeEach(func() {
```

Comment:
This is a question, not suggestion - there is no way to make this public to the package and not have to redefine it in each test? It is the same as the one in `predicates_test.go`

Reply:
This was sort of the same question David made to me but in the opposite direction :D. He was concerning that some constants I had defined globally would eventually clash with other parts of the `tekton` package, therefore I moved it into the function.

Reply:
@davidmogar thoughts?

Reply:
Sharing variables in code is ok but not a fan of sharing them in tests. Nevertheless I don't have an strong opinion on this. Your call Leandro.

Reply:
Thanks for the feedback @davidmogar and @johnbieren. I would prefer keep it like this for now. If it become necessary (as tests tends to increase) I might refactor it.